### PR TITLE
Added the class method `above_average_rating`, included with the `act…

### PR DESCRIPTION
**Pull request by antonivanopoulos**
…s_as_rateable` module. Returns all objects of the rateable type above or equal to the average rating supplied.

### CODE REVIEW

File: lib/is_rateable/acts_as_rateable.rb [hunk 1]
```diff
@@ -11,6 +11,9 @@ def acts_as_rateable(options = {})
 
         has_many :ratee_ratings, as: :ratee, class_name: 'IsRateable::Rating'
 
+        def above_average_rating(rating)
```
Comment (RustComet) [inline lib/is_rateable/acts_as_rateable.rb]: @antonivanopoulos Should this be down in the `LocalInstanceMethods`?

Comment (antonivanopoulos) [inline lib/is_rateable/acts_as_rateable.rb]: @RustComet It's a class method / scope that returns the collection of objects above the supplied rating, i.e. `Movie.above_average_rating(5)`

`LocalInstanceMethods` is only for instance methods, i.e. `@movie.stuff`.

Comment (RustComet) [inline lib/is_rateable/acts_as_rateable.rb]: @antonivanopoulos Haha. Oops. Of course it is. My bad. 
:shipit:

Comment (antonivanopoulos) [inline lib/is_rateable/acts_as_rateable.rb]: @RustComet Haha all good :tada:

**Pull request merged**